src/presentation/buildQuizPage.ts

```diff
@@ -1,25 +1,31 @@
 import { Question } from "../questions/types";
-import { QuizMaster } from "./QuizMaster";
-import { fillTemplate } from "./fetchTemplate";
-import { displayNextButton, displaySubmitButton } from "./addButton";
+import { QuizMaster } from "../game/QuizMaster";
+import { FetchTemplate } from "./FetchTemplate";
+import { AddButton } from "./AddButton";
+import { Quiz } from "../game/Quiz";
 
 window.onload = function () {
-    buildQuizPage();
+    const quizMaster = new QuizMaster();
+    let quiz = quizMaster.newQuiz();
+    
+    let storageScore = localStorage.getItem("quizScore");
```

Comment:
The score is part of the quiz. Believe me, jut take a look into your quiz class. ;)